mesheryctl/internal/cli/root/model/model.go

```diff
@@ -276,35 +282,129 @@ func exportModel(modelName string, cmd *cobra.Command, url string, displayCountO
 	data, err := io.ReadAll(resp.Body)
 	if err != nil {
 		utils.Log.Error(err)
-		return err
+		return nil
 	}
 
 	modelsResponse := &models.MeshmodelsAPIResponse{}
 	err = json.Unmarshal(data, modelsResponse)
 	if err != nil {
 		utils.Log.Error(err)
-		return err
+		return nil
 	}
 	if len(modelsResponse.Models) < 1 {
-		return ErrExportModel(fmt.Errorf("Model with the given name could not be found in the registry"), modelName)
+		return ErrExportModel(fmt.Errorf("model with the given name could not be found in the registry"), modelName)
 	}
 	model := modelsResponse.Models[0]
+	_ = model.ReplaceSVGData("../")
```

Comment:
Will you add a comment explaining this?

Are we writing to "../" or reading files from "../"?

Reply:
Sure.